test/test_io.cpp

```diff
@@ -13,16 +13,16 @@
 #include <simulation.hpp>
 namespace fs = std::filesystem;
 
+using namespace Seldon;
+using namespace Catch::Matchers;
+using AgentT  = ActivityDrivenModel::AgentT;
+using Network = Network<AgentT>;
+
+auto proj_root_path = fs::current_path();
```

Comment:
We don't have conventions for this, but it seems like this should be `const` and be in caps as a global variable.?
```suggestion
const std::filesystem::path CURRENT_PATH = std::filesystem::current_path();
```

Also this will only be proj_root_path if it is run from the project root... So maybe even...